src/extension/_locales/en/messages.json

```diff
@@ -137,6 +137,30 @@
     "close_sidekick": {
       "message": "Close Sidekick"
     },
+    "column_name_url": {
+      "message": "url"
```

Comment:
A translator might be inclined to make acronyms like this uppercase. If it should remain lowercase, we should add a `description` property explaining that.